code/modules/assembly/flash.dm

```diff
@@ -24,7 +24,8 @@
 
 /obj/item/flashbulb/examine(mob/user)
 	. = ..()
-	. += "This one can probably just about handle [charges_left] more uses."
+	. += "[charges_left == 0 ? "The bulb is completely burnt out" : charges_left == 1 ? "This bulb will probably burn out immediately in a flash" : "This bulb can probably just about handle [charges_left] more uses."]"
```

Comment:
"This bulb will probably burn out on its next use"
Also add a period to all the options.